tests/manual/test-procedure.rst

```diff
@@ -446,6 +446,23 @@ Test Case: Autocomplete
 - [ ] Disable autocomplete in "Options" and confirm that autocomplete no longer offers suggestions.
 
 
+Test Case A11y: Keyboard focus follows a logical order
+''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Using your keyboard, navigate through the menu and any submenus using the `tab` and `enter` or `space` keys.
+- [ ] Using your keyboard, navigate through the modal windows using the `tab` and `enter` or `space` keys.
+
+
+Test Case A11y: Esc key returns focus to menu
+'''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Type something un the text window, then confirm the `Esc` key returns the focus to the menu from the text window.
+- [ ] Enter a modal window and confirm the `Esc` key closes an open modal.
```

Comment:
Just to be clear we should pick one modal and explicitly indicate to open that one.

Reply:
done in https://github.com/bbcmicrobit/PythonEditor/pull/304/commits/583c086845ced339529c69442d57a186387ff109